.travis.yml

```diff
@@ -41,19 +41,27 @@ install:
       else
         wget -q https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh -O miniconda.sh;
       fi
+    # Necessary to install pyNFFT
+    - if [ $TRAVIS_PYTHON_VERSION == "3.6" ]; then
+        export CPLUS_INCLUDE_PATH=/opt/python/3.6.3/include/python3.6m;
```

Comment:
I don't understand what you mean.

Reply:
I guess what he meant was that it spells out the path to Python and the exact version number, which will likely break when a new patch version comes out.
Perhaps there is a Travis environment variable for this?

Reply:
I don't think there is a Travis env var for this (cf [this](https://docs.travis-ci.com/user/environment-variables/#default-environment-variables)). I see however that you can [specify the patch](https://github.com/travis-ci/travis-ci/issues/7796) in the python version. Maybe it would be a good idea to set it, since after all, potentially other things might break with another patch change.